src/Symfony/Component/Routing/Tests/Fixtures/AttributesFixtures/AttributesClassParamAfterCommaController.php

```diff
@@ -3,14 +3,13 @@
 namespace Symfony\Component\Routing\Tests\Fixtures\AttributesFixtures;
 
 use Symfony\Component\Routing\Tests\Fixtures\Attributes\FooAttributes;
-use Symfony\Component\Security\Core\User\User;
 
 #[FooAttributes(
     foo: [
         'bar' => ['foo','bar'],
         'foo'
     ],
-    class: User::class
+    class: \stdClass::class
```

Comment:
We might want to backport those fixture changes.

Reply:
done in #40463